ouroboros-consensus/test/infra-test/Test/Ouroboros/Consensus/PointSchedule.hs

```diff
@@ -366,6 +390,28 @@ splitFragmentAtSlotNo _ (Empty anchor) =
   (Empty anchor, Empty anchor)
 
 -- | Returns the block number of the given slot.
--- REVIEW: might just be 'blockSlot'?
+-- REVIEW: might just be 'blockSlot'? yes!
 blockSlotNo :: HasHeader b => b -> WithOrigin SlotNo
 blockSlotNo = getTipSlotNo . tipFromHeader
+
+data ScheduleType =
```

Comment:
Here or elsewhere it needs to be documented what FastAdversary and Banal types of schedule are.